api/e2etests/e2e_seed_data.py

```diff
@@ -34,11 +35,41 @@ def seed_data() -> None:
     )
     org_admin.add_organisation(organisation, OrganisationRole.ADMIN)
 
-    project: Project = Project.objects.create(
-        name="My Test Project", organisation=organisation
-    )
-    Environment.objects.create(name="Development", project=project)
-    Environment.objects.create(name="Production", project=project)
+    project_test_data = [
+        {
+            "name": "My Test Project",
+            "environments": [
+                "Development",
+                "Production",
+            ],
+        },
+        {"name": "My Test Project 2", "environments": ["Development"]},
+        {"name": "My Test Project 3", "environments": ["Development"]},
+        {"name": "My Test Project 4", "environments": ["Development"]},
+    ]
+
+    # Create projects and environments
+    projects = []
+    environments = []
+    for project_info in project_test_data:
+        project = Project.objects.create(
+            name=project_info["name"], organisation=organisation
+        )
+        projects.append(project)
+
+        for env_name in project_info["environments"]:
+            environment = Environment.objects.create(name=env_name, project=project)
+            environments.append(environment)
+
+    # Create identities
+    identities_test_data = [
+        {"identifier": settings.E2E_IDENTITY, "environment": environments[2]},
+        {"identifier": settings.E2E_IDENTITY, "environment": environments[3]},
+        {"identifier": settings.E2E_IDENTITY, "environment": environments[4]},
+    ]

```

Comment:
We should add a comment explaining why we're only creating identities for 3 of the 5 environments.

Reply:
Added.